p4-16/spec/grammar.mdk

```diff
@@ -541,7 +550,7 @@ headerStackType
 
 // BEGIN:specializedType
 specializedType
-    : prefixedType "<" typeArgumentList ">"
+    : typeName "<" typeArgumentList ">"
```

Comment:
typeName's only production rule is to expand to prefixedType, so this is not a change in syntax.  The p4c grammar uses the version with typeName.